packages/app-mobile/components/screens/ConfigScreen/plugins/PluginBox.tsx

```diff
@@ -37,6 +37,26 @@ interface Props {
 	onShowPluginLog?: PluginCallback;
 }
 
+// defines the recommended chip press event alert
+// moved to top level to avoid redefining the function on each render since this isn't plugin specific
+// doing so eliminates the need to use useCallback
+const onRecommendedPress = () => {
+	Alert.alert(
+		_(''),
```

Comment:
The title is a required field, so bypassed it by using an empty string, as recommended [here](https://reactnative.dev/docs/alert#alert)